ui/imodel-components-react/src/imodel-components-react/timeline/SolarTimeline.scss

```diff
@@ -11,14 +11,15 @@ $bottom-position: 4px;
   height: 28px;
 }
 
+// TODO: Can we get rid of this wrapper?
 .solar-timeline-wrapper {
```

Comment:
I think we can&should merge `.solar-timeline-wrapper` & `.header`.

Reply:
Done!